Source/ProjectFirma.Web/Views/DNRUplandRegion/Detail.cshtml

```diff
@@ -118,6 +120,68 @@
         </div>
     </div>
 </div>
+<div class="row">
+    <div class="col-md-5">
```

Comment:
Have some ideas for this page. I think we should make the map the full width. Then put the Contact info section below that on the left and the Grant Allocation Expenditures by Cost Type section to the right. 
This would give them a bigger map and I think a cleaner looking page.

Reply:
Great idea!